check_manifest.py

```diff
@@ -34,7 +34,12 @@
 from typing import List, Optional, Union
 from xml.etree import ElementTree as ET
 
-import tomli
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
Whee, now we have test coverage enforcement issues!

Both branches need a `# pragma: nocover`

Reply:
Updated!